### ISSUE TITLE
Gradient Clipping blow up

### ISSUE THREAD

**Issue by jacenfox** [MathGaron/pytorch_toolbox#23]
https://github.com/MathGaron/pytorch_toolbox/blob/6e6c88f8e3eb7259763d7215cd9c3e614d065507/pytorch_toolbox/train_loop.py#L147

Be careful about this operation, it can blow up your training, I don't know when, but it doesn't work on my LSTM network. 
Maybe it's better to set up the threshold manually.

**Comment by MathGaron** [comment]
It is not set by default! and for the value, I am not convinced that it is useful at all to let the user chose a value? 1 seems pretty logical, I want to be sure that it is useful to set it up to other values before adding more hyperparameters!
